src/styles/index.css

```diff
@@ -0,0 +1,161 @@
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  height: auto;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 0;
+  background-image: url(../images/imac.jpeg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.card__title {
+  padding: 0 16px;
+  margin-top: 40px;
+  line-height: 18px;
+  height: 36px;
+  width: 166px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
+}
+
+.card__product-code {
+  box-sizing: border-box;
+  margin-top: 4px;
+  padding: 0 16px;
+  width: 100%;
+  justify-content: center;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__reviews {
+  display: flex;
+  margin: 16px;
+  margin-bottom: 0;
+  width: 166px;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  border-radius: 0.5px;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(../images/Star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--4 .stars__star:last-child {
+  margin-right: 0;
+}
+
+.reviews {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+  align-self: center;
+}
+
+.card__price {
+  display: flex;
+  width: 166px;
+  line-height: 18px;
+  justify-content: space-between;
+  margin: 24px 16px 0;
+  opacity: 1;
+}
+
+.price {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: #616070;
+}
+
+.amount {
+  font-family: Roboto, sans-serif;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.button {
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  width: 166px;
+  height: 40px;
+  margin: 16px;
+  border-radius: 5px;
+  opacity: 1;
+  background: #00acdc;
+}
+
+.button__buy {
+  text-decoration: none;
+  text-transform: uppercase;
+  color: white;
+  font-family: Roboto, sans-serif;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+}
+
+.button:hover {
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  width: 166px;
+  height: 40px;
+  margin: 16px;
+  border-radius: 5px;
+  opacity: 1;
+  border: 1px solid #00acdc;
+  background: white;
```

Comment:
The hover state for the button repeats many properties from the default state. Consider only specifying the properties that change on hover to reduce redundancy and improve maintainability.